terraform/gcp-kubernetes-cluster/cluster.tf

```diff
@@ -0,0 +1,89 @@
+resource "google_container_cluster" "cluster" {
+  name                     = "${var.cluster_name}-cluster"
+  location                 = var.location_1
+  remove_default_node_pool = true
+  initial_node_count       = 1
+  network                  = google_compute_network.network.self_link
+  subnetwork               = google_compute_subnetwork.subnetwork.self_link
+  networking_mode          = "VPC_NATIVE"
+  deletion_protection      = false
+  # logging_service          = "logging.googleapis.com/kubernetes"
+  # monitoring_service       = "monitoring.googleapis.com/kubernetes"
+
+  # Optional, if you want multi-zonal cluster
+  node_locations = [
+    "${var.location_2}"
+  ]
+
+  addons_config {
+    http_load_balancing {
+      disabled = true # NGINX Ingress
+    }
+
+    horizontal_pod_autoscaling {
+      disabled = false # Self-managed
+    }
+  }
+
+  release_channel {
+    channel = "RAPID"
+  }
+
+  workload_identity_config {
+    workload_pool = "${var.project_id}.svc.id.goog"
+  }
+
+  ip_allocation_policy {
+    cluster_secondary_range_name  = "${var.cluster_name}-pod-range"
+    services_secondary_range_name = "${var.cluster_name}-service-range"
+  }
+
+  private_cluster_config {
+    enable_private_nodes    = true
+    enable_private_endpoint = false
+    master_ipv4_cidr_block  = "172.16.0.0/28"
+  }
+
+  # Enable Kubernetes Gateway API
+  gateway_api_config {
+    channel = "CHANNEL_STANDARD"
+  }
+
+  #   Jenkins use case
```

Comment:
we won't use Jenkins so this isn't needed?

Reply:
No, I will remove it

Reply:
Done